src/software/ai/hl/stp/play/free_kick_play_test.py

```diff
@@ -0,0 +1,258 @@
+import sys
```

Comment:
@itsarune The AI in these free kick play tests has some interesting behaviour. The robots sometimes drive across the field to take the free kick when there's already a robot by the ball. For some, the enemy seems to try to get the ball before(?) the ball is kicked. If these issues aren't tracked, it would be worth adding new issues for them and tagging them in this test/in the play. 
(note that I'm running with the latest changes from master pulled)

Reply:
yeah the tactic assignment stuff may not be fixable beyond updating the cost functions but we have a PR that merges free kick play with offence play, so we may not need to worry about free kick behavior

Reply:
Are you referring to an upcoming PR, or #2695? If you're referring to #2695, then do we have a plan of getting that cleaned up and merged in (probably should be closed and opened by someone else so we can update it easily)?

Reply:
I'm referring to #2695, it's part of the task list for the Gameplay project, but we haven't gotten to cleaning it up yet

Reply:
Sounds good. We should consider closing the PR in the meantime then.